azure-pipelines.yml

```diff
@@ -39,7 +39,7 @@ steps:
   inputs:
     command: 'test'
     projects: '$(solutionDirectory)\Selenium.Axe.Test\Selenium.Axe.Test.csproj'
-    arguments: '--collect:"Code Coverage"'
+    arguments: '--no-build --collect:"Code Coverage"'
```

Comment:
This will only work if you add 
> -c $(buildConfiguration)

to the arguments. 
The issue is that the CLI assumes a different (debug) build type.